Jenkinsfile

```diff
@@ -79,6 +79,7 @@ pipeline {
       steps{
         sh 'ci/submit-coverage'
         archiveArtifacts artifacts: "coverage/.resultset.json", fingerprint: false
+        archiveArtifacts artifacts: "ci/authn-k8s/output/simplecov-resultset-authnk8s-gke.json", fingerprint: false
```

Comment:
You want the `archiveArtifacts` for the gke output to come before `sh 'ci/submit-coverage'` since Line 80 is where the failure was occurring.